IPFS_ALL_HANDS_FORMAT.md

```diff
@@ -45,18 +45,36 @@ If you have a talk you want to give, [sign up here](https://docs.google.com/spre
 ![](https://res.cloudinary.com/blockchain-side-hustle/image/upload/v1542475278/ipfs_calendar_dyxtst.jpg)
 
 
+## Steps to signing up and speaking
 
-_[Sign up here if you would like to speak!](https://docs.google.com/spreadsheets/d/1XRB2QsPzCPLPOErKvDZfOKK3CMohI9t_QKNdztYMlK0/edit#gid=350755898)_
+1. Write a short blurb about your talk and a proposed date for your talk. Send it to portia @ protocol dot com. You should receive an update about the status of your talk within 24-48 hours.
```

Comment:
@pkafei you got your email wrong again. I'm starting to think that you are just trolling me :)